google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -147,6 +164,7 @@ appengine.run.settings.apiport.label=API port:
 appengine.run.settings.loglevel.label=Application log level:
 appengine.run.settings.adminhost.label=Admin host:
 appengine.run.settings.adminport.label=Admin port:
+appengine.infopanel.noproject=Can't retrieve region for selected project.
```

Comment:
Yes, it's different. This is e.g., for when a project doesn't exist. It's shown as an error. You mean I should add error somewhere in the description?